src/app/core/services/localeService.ts

```diff
@@ -1,24 +1,41 @@
 import { Injectable } from '@angular/core';
-import { appConfig } from 'src/app/shared/helpers/config';
+import { appConfig, LanguageConfig } from 'src/app/shared/helpers/config';
+
 
 @Injectable({
   providedIn: 'root'
 })
 export class LocaleService {
 
-  readonly languages?: { tag: string, path: string }[];
-  readonly currentTag?: string;
+  readonly languages: LanguageConfig[];
+  readonly currentTag: string;
+  readonly currentTagError: boolean
 
   constructor() {
     this.languages = appConfig.languages;
-    this.currentTag = this.languages?.find(l => window.location.pathname.endsWith(l.path))?.tag;
+    const fallbackTag = appConfig.production ? '' : 'en';
+    this.currentTag = this.languages.find(l => window.location.pathname.endsWith(l.path))?.tag ?? fallbackTag;
```

Comment:
Adding the language config `{ path: '/', tag: 'en' }` does not work because the path `"/"` matches every algorea url, it will fallback to english every time the url language is none of the configured ones. There will never be any issue with that config but the language error will be silent.
Plus, it adds a tricky situation because since it matches every url, it must stay at the end of the config languages array.

Since I considered that matching `"/"` was not a solution, I needed to handle the development use-case.
Maybe it will be easier to talk about it :sweat_smile:

Reply:
Another solution is to change the language matching logic: instead of using `pathname.endsWith(langTag)`, we would have to match it entirely: `pathname === langTag`. But I don't know if that would be a breaking change in production.

Reply:
Fixed